pkg/scheduler/eventhandlers_test.go

```diff
@@ -334,3 +343,103 @@ func TestPreCheckForNode(t *testing.T) {
 		})
 	}
 }
+
+// test for informers of resources we care about is registered
+func TestAddAllEventHandlers(t *testing.T) {
+	tests := []struct {
+		name                   string
+		gvkMap                 map[framework.GVK]framework.ActionType
+		expectStaticInformers  map[reflect.Type]bool
+		expectDynamicInformers map[schema.GroupVersionResource]bool
+	}{
+		{
+			name:   "default handlers in framework",
+			gvkMap: map[framework.GVK]framework.ActionType{},
+			expectStaticInformers: map[reflect.Type]bool{
```

Comment:
Usually in golang, we use `map[T]stuct{}` to represent a "Set". But it's just a nit.

Reply:
to change this, we would have to change the informer's code. This is fine for our purposes.